### PR TITLE
Add dev.geopjr.Collision

### PR DESCRIPTION
**Pull request by GeopJr**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

### Info

- This is an app ID change of `dev.geopjr.Hashbrown` ([Collision#26](https://togithub.com/GeopJr/Collision/issues/26) & [Collision#27](https://togithub.com/GeopJr/Collision/issues/27)). EOL is drafted https://github.com/flathub/dev.geopjr.Hashbrown/pull/5.

- Since 42 is not available yet, gtk & pongo get built for libadwaita 1.0.2 (the app depends on some style classes). They'll get removed when 42 becomes available!

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by GeopJr** [comment]
bot, build dev.geopjr.Collision

File: dev.geopjr.Collision.yaml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      - type: git
+        url: https://github.com/libevent/libevent.git
+        tag: release-2.1.12-stable
+  # Required by libadwaita
+  - name: libsass
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/lazka/libsass.git
+        branch: meson
```
Comment (hfiguiere) [inline dev.geopjr.Collision.yaml:33]:
```suggestion
```

File: dev.geopjr.Collision.yaml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        commit: 302397c0c8ae2d7ab02f45ea461c2c3d768f248e
+  # Required by libadwaita
+  - name: sassc
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/lazka/sassc.git
+        branch: meson
```
Comment (hfiguiere) [inline dev.geopjr.Collision.yaml:41]:
```suggestion
```

File: flathub.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: this is really unfortunate

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by barthalion** [comment]
bot, build dev.geopjr.Collision

**Comment by barthalion** [comment]
bot, build dev.geopjr.Collision

**Comment by GeopJr** [comment]
bot, build dev.geopjr.Collision

File: flathub.json (hunk 3 above)
Comment (GeopJr) [reply flathub.json:2]: I'd rather wait until it gets Tier 1, but will think about it!

(https://crystal-lang.org/reference/1.3/platform_support.html#tier-2)
> Tier 2 platforms can be thought of as “expected to build”. Automated tests are not run so it’s not guaranteed to produce a working build, but platforms often work to quite a good degree and patches are always welcome!

**Pull request closed without merging**